simularium/simularium-website
# website patch for SelectionStateInfo changes

Time estimate or Size
_small_

Problem
=======
Viewer release v3.8.4 includes breaking changes to `SelectionStateInfo` from this [PR #400](https://github.com/simularium/simularium-viewer/pull/400).
Patching advances https://github.com/simularium/simularium-website/issues/510 and https://github.com/simularium/simularium-website/issues/511 

Solution
========
This patch:
- installs the newly released viewer
- removes individual colorChange from redux as it is no longer part of `SelectionStateInfo`
- updates the selector `getSelectionStateInfoForViewer` to instead use the current `UIDisplayData` as the value for `appliedColors`


Steps to Verify:
----------------
1. Try changing some colors to make color picker is not broken!
2. Verify that side panel and rendered scene are in sync.
